### ISSUE TITLE
🟢[SDXL FEATURE REQ] EQ VAEs and padding patch [sd3 branch]

### ISSUE THREAD

**Issue by deGENERATIVE-SQUAD** [kohya-ss/sd-scripts#2189]
We have two branches of new, especially precise and “clean” VAEs for SDXL called EQ VAE (one from https://huggingface.co/KBlueLeaf/EQ-SDXL-VAE ). Test trainings show a visually noticeable improvement when using these new VAEs for training.

Problems:

1. Anzhc’s EQ VAE (https://huggingface.co/Anzhc/MS-LC-EQ-D-VR_VAE) loads correctly, but also requires the patch from the project page to dodge the padding bug:

```
for module in self.model.modules():
            if isinstance(module, nn.Conv2d):
                pad_h, pad_w = module.padding if isinstance(module.padding, tuple) else (module.padding, module.padding)
                if pad_h > 0 or pad_w > 0:
                    module.padding_mode = "reflect"
```

Additional module for the inference fix via Comfy with models trained on Anzhc’s EQ VAE here https://github.com/Jelosus2/comfyui-vae-reflection

It would be great to be able to use in the full extent, if that’s not too much trouble to implement.
